src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4447,6 +4447,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
  /fullpaths                    Compiler generates fully qualified paths
  /filealign:&lt;n&gt;                Specify the alignment used for output file 
                                sections
+ /pathmap:&lt;K1&gt;=&lt;V1&gt;,&lt;K2&gt;=&lt;V2&gt;,...      Specify a mapping for source path names output by
```

Comment:
I will.

Please use the word "consider" as a suggestion to think about a possible change. If you think the change is mandatory (as I believe this one is), the primary verb should be "Wrap" rather than "Consider".